### DISCUSSION TITLE
Unable to play youtube videos in Hubs

### DISCUSSION THREAD

**Discussion by Kanishk-Malick** [mozilla/hubs#2929] [General Questions & Discussion] [answered]
Getting this error : 
Unable to load video
we have exceeded our quota on resolving videos from this provider. Please try again later

**Comment by cvanmeurs** [comment, marked as answer]
This is a frequently reported issue.  Please refer to the following FAQ:

https://github.com/mozilla/hubs/discussions/2681

**Reply by Kanishk-Malick** [reply]
That's terrible.

**Reply by emclaren** [reply]
We can't control third party services and their limits, which is why we recommend hosting videos directly on Hubs or via storage that you control
